README.md

```diff
@@ -39,13 +39,15 @@ Things to be aware of:
 
 1. [Install Docker](https://www.docker.com/get-started)
 2. Clone the repo
-3. In the root directory, run `npm install`
+3. In the frontend root directory, run `npm install`
+4. Bonus: probably want to install react and redux developer tools extensions
+5. Bonus2: probably want the api service up or this won't work
```

Comment:
yeah...i forgot to add this to the docs. The README is already out-of-date and poorly maintained.

Reply:
I'll update the docs in another PR with instructions no how to do this and where to find this